src/stylesheets/components/_contact-panel.scss

```diff
@@ -0,0 +1,24 @@
+.app-c-contact-panel {
+  @include govuk-responsive-margin($govuk-spacing-responsive-6, "top");
+  @include govuk-responsive-padding($govuk-spacing-responsive-3);
+  background-color: $govuk-blue;
+}
+
+.app-c-contact-panel--heading {
+  @extend .govuk-heading-m;
```

Comment:
I don't think we should be using extends like this, but this seems like a common need we need to address. We should possibly look at providing mixins for these typography classes you can `@include` instead.

Reply:
Could consider just using the concrete class in the markup too

Reply:
I might leave this one as is unless there's any strong objection?